.circleci/config.yml

```diff
@@ -42,6 +42,12 @@ commands:
       - run:
           name: Test kubectl
           command: kubectl
+      - kube-orb/install-kubeconfig:
+          kubeconfig: dGVzdA==
```

Comment:
Could you update this step since the `kubeconfig` parameter is now an environment variable name? I'll add a KUBECONFIG environment variable for this project.